bpf/common/ssl_helpers.h

```diff
@@ -73,6 +73,6 @@ connect_ssl_to_connection(u64 id, pid_connection_info_t *conn, u8 direction, u16
     set_active_ssl_connection(&ssl_conn, ssl);
 }
 
-static __always_inline void *is_ssl_connection(pid_connection_info_t *conn) {
-    return bpf_map_lookup_elem(&active_ssl_connections, conn);
+static __always_inline u64 *is_ssl_connection(pid_connection_info_t *conn) {
```

Comment:
Having this as void * caused a nasty bug, we were getting technically a pointer to the u64, but we were later querying with & of the result of this call, instead of the value directly.